rave-compiler/src/main/java/com/uber/rave/compiler/RaveWriter.java

```diff
@@ -97,6 +99,9 @@ public void generateJava(@NonNull RaveIR raveIR) throws IOException {
         String className = raveIR.getSimpleName() + GENERATED_CLASS_POSTFIX;
         TypeSpec.Builder builder = TypeSpec.classBuilder(className);
         builder.superclass(BaseValidator.class)
+                .addAnnotation(AnnotationSpec.builder(Generated.class)
+                        .addMember("value", "RaveProcessor")
```

Comment:
> Make this the fully qualified name of the code generator as recommended in the docs.

9f464b0

> A comment indicating this validator is generated for all models annotated with @Validated in this module/library would also be helpful.

No. It'd be redundant at most since that's what the library documentation is for. Instead, I just matched what dagger does (768f4f8)